cli-tests/cli-tests.bats

```diff
@@ -354,8 +360,7 @@ aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaa\
 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaa\
 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaaaaaaaaaaa
   id=$(bupstash put :: "$SCRATCH/foo")
-  bupstash get id=$id | tar -tf - | wc -l
```

Comment:
Is this one OK to remove?

Reply:
I think so, this might have just been leftover from some debugging.